src/ome_arrow/tensor.py

```
@@ -0,0 +1,868 @@
+            device: Target device ("cpu" or "cuda").
+            contiguous: When True, materialize a contiguous buffer if needed.
+            mode: Export mode. "arrow" returns a 1D values buffer.
+
+        Returns:
+            jax.Array: Array backed by the DLPack capsule.
+        """
+
+        try:
+            import jax
+        except ImportError as exc:
+            raise RuntimeError("JAX is not installed.") from exc
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)